quickstart.org

```diff
@@ -55,57 +82,97 @@ To buy:
 #+BEGIN_SRC org
 - [ ] do
 - [-] the
 - [X] thing
 #+END_SRC
-* Images
-Just link to an image, and don't give a description. Seriously, that's it. \\
-~[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]~
 
-#+attr_html: :height 120px
-[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]
+Lists can contain tags (and checkboxes at the same time).
+#+BEGIN_SRC org
+- [ ] fruits :: get apples
+- [X] veggies :: get carrots
+#+END_SRC
+
 * Links
-Just put a target in between two square brackets, like so: ~[[target]]~. Want to
-add a description? Just add a ~[description]~ after the target, i.e.
-~[[target][desc]]~.
+To create a link put the target between two square brackets, like so: ~[[target]]~.
+To add a description add it in square brackets after the target ~[[target][desc]]~.
+
+The order is easy to remember because it is the same as an html =<a>=
+tag ~<a href="target">desc</a>~ and you can't mix up brackets and
+parentheses because there are only square brackets!
 
-Org can handle a variety of different types of targets, which you use by giving
-~[target]~ as ~[type:target]~. When no ~type:~ is provided, org just searches for a
-heading which matches ~target~.
+Org supports a wide variety of types of link targets, and you can also
+[[https://orgmode.org/manual/Adding-Hyperlink-Types.html][add your own]].
+Link types are indicated with a ~type:~ prefix, e.g. ~[type:target]~.
+If no ~type:~ is provided, org searches the current file for a headline
+that matches ~target~.
 
-A few examples:
+Some examples:
 #+BEGIN_EXAMPLE
 [[https://orgmode.org][a nice website]]
 [[file:~/Pictures/dank-meme.png]]
 [[earlier heading][an earlier heading in the document]]
 #+END_EXAMPLE
 
 ** More info
-For more information, see the manual sections on [[https://orgmode.org/manual/Internal-Links.html][Internal Links]] and [[https://orgmode.org/manual/External-Links.html#External-Links][External Links]].
-* Code blocks
-To just have a monospace block,
+For more on links, see the manual sections for
+[[https://orgmode.org/manual/Internal-Links.html][Internal Links]] and
+[[https://orgmode.org/manual/External-Links.html][External Links]].
+* Images
+Org mode automatically recognizes and renders image links.
+Just link to an image (don't include a description). That's it. \\
+~[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]~
+
+#+attr_html: :height 120px
+[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]
 
+* Blocks
+Org mode uses =#+BEGIN= ... =#+END= blocks for many purposes.
+Some of the basic blocks types quote, example, and src.
+
+If you all you need is monospace text, you can use an example block.
 #+BEGIN_SRC org
 ,#+BEGIN_EXAMPLE
 monospace
 ,#+END_EXAMPLE
 #+END_SRC
 
-If you want syntax highlighting and execution try
+However, example blocks do not give you the awesome power of [[https://orgmode.org/worg/org-contrib/babel/][Org babel]].
```

Comment:
Personal taste, but how about "phenomenal" over "awesome"?

Reply:
I don't think that phenomenal works in this context. Unfortunately it does evoke the "Like 10,000 hotdogs sir." definition of awesome. Maybe just "do not give you the power of"?

Reply:
That seems like the best approach to me :slightly_smiling_face: I just didn't quite feel that "awesome" fit that well.